build.ps1

```diff
@@ -552,7 +552,7 @@ End {
                 Write-Output "      osVersion: $($tag.osVersion)"
                 Write-Output "      architecture: $($tag.architecture)"
                 Write-Output "      os: $($tag.os)"
-                Write-Output "      dockerfile: $($tag.dockerfile)"
+                Write-Output "      Dockerfile: $($tag.dockerfile)"
```

Comment:
please revert.  Capitalization matters in code and I don't own the service that consumes this.